typescript/src/chain.ts

```diff
@@ -43,12 +43,14 @@ export interface Bridge {
    * @param depositOutputIndex - Index of the deposit transaction output that
    *        funds the revealed deposit
    * @param deposit - Data of the revealed deposit
+   * @returns Transaction hash of the reveal deposit transaction as string
```

Comment:
Just flagging: The type of this parameter is an important design decision in the context of potential multi-chain support. A `string` makes sense for Ethereum but may be not suitable for other chains. That said, I think we are good for now but let's keep that in mind while making that change for other methods of the `Bridge` interface.